app/src/views/Home.vue

```diff
@@ -1,18 +1,247 @@
+              <b-dropdown-item>Second Action</b-dropdown-item>
+              <b-dropdown-item>Third Action</b-dropdown-item>
+              <b-dropdown-divider></b-dropdown-divider>
+              <b-dropdown-item active>Active action</b-dropdown-item>
+              <b-dropdown-item disabled>Disabled action</b-dropdown-item>
+            </b-dropdown>
+          </b-col>
+        </b-row>
+      </div>
+      <br />
+      <!-- #endregion -->
+      <!-- #region If there less tags than notes then some are untagged. They can have a special section -->
+      <b-row v-if="tags.length < data.length">
+        <b-col class="col"><hr /></b-col>
+        <b-col class="col-auto">Untagged</b-col>
+        <b-col class="col"><hr /></b-col>
+      </b-row>
+      <b-row>
+        <b-col sm="3" v-for="item in data" :key="item.title">
+          <b-card
+            class="m-1"
+            v-if="!item['tag']"
+            :title="item.note['title']"
+            align="left"
+          >
+            <b-card-text>
+              <b-card-text class="small text-muted">
+                <b-icon icon="clock-fill"></b-icon>
+                Last updated
+                {{ timeSince(new Date(item.note["changed"])) }}
+                ago</b-card-text
+              >
+            </b-card-text>
+          </b-card>
+        </b-col>
+      </b-row>
+      <br />
+      <!-- #endregion-->
+      <!-- #region Make a header for every tag and fill in the notes that belong to it -->
+      <div v-for="tag in tags" :key="tag">
+        <b-row>
+          <b-col class="col"><hr /></b-col>
+          <b-col class="col-auto">{{ tag }}</b-col>
+          <b-col class="col"><hr /></b-col>
+          <br />
+        </b-row>
+        <b-row class="justify-content-start">
+          <b-col
+            sm="3"
+            v-for="item in data.filter((x) => x['tag'] == tag)"
+            :key="item.title"
+          >
+            <b-card
+              class="m-1"
+              v-if="item['tag'] == tag"
+              :title="item.note['title']"
+              align="left"
+            >
+              <b-card-text>
+                <b-card-text class="small text-muted">
+                  <b-icon icon="clock-fill"></b-icon>
+                  Last updated
+                  {{ timeSince(new Date(item.note["changed"])) }}
+                  ago</b-card-text
+                >
+              </b-card-text>
+            </b-card>
+          </b-col>
+        </b-row>
+        <br />
+      </div>
+      <!-- #endregion -->
+    </div>
+    <!-- #endregion -->
   </div>
 </template>
 
 <script>
 // @ is an alias to /src
-import HelloWorld from '@/components/HelloWorld.vue'
+import { BIcon } from "bootstrap-vue";
+import "vue-sidebar-menu/dist/vue-sidebar-menu.css";
+import { SidebarMenu } from "vue-sidebar-menu";
 
 export default {
-  name: 'Home',
+  name: "Home",
   components: {
-    HelloWorld
-  }
-}
+    BIcon,
+    SidebarMenu,
+  },
+  data() {
```

Comment:
What is this section containing? Should have a comment

Reply:
is this just filler stuff since nothing is hooked up to MySky yet?

Reply:
It's part of the Vue template. Every page has a data function which stores data related to that page